frontend/src/features/map_planning/layers/base/BaseLayer.tsx

```diff
@@ -46,6 +71,23 @@ const BaseLayer = (props: BaseLayerProps) => {
           offset={imageOffset}
         />
       )}
+      {untrackedBaseLayerState.measurePoint1 && (
+        <Circle
+          x={untrackedBaseLayerState.measurePoint1.x}
+          y={untrackedBaseLayerState.measurePoint1.y}
+          radius={editorLongestSide / 250}
+          fill="red"
+        />
+      )}
+      {untrackedBaseLayerState.measurePoint2 && (
+        <Circle
+          x={untrackedBaseLayerState.measurePoint2.x}
+          y={untrackedBaseLayerState.measurePoint2.y}
+          radius={editorLongestSide / 250}
+          fill="red"
```

Comment:
You could try using the primary color here instead of red to better fit the design. If the contrast is too bad then leave it red as it is easy to spot.

Reply:
I exchanged the full bright red for a less saturated shade of the same color.
The primary color is hard to differentiate from green surfaces on the base layer.